.github/workflows/smoke_test.yml

```diff
@@ -0,0 +1,38 @@
+name: Smoke Test
+on:
+  workflow_dispatch:
+    inputs:
+      release:
+        type: string
+        description: Release number
+        required: true
+
+jobs:
+  build_cli:
+    name: Build cli from release ${{ github.event.inputs.release }}
+    runs-on: public-amd64-2xlarge
+    steps:
+      - name: Get Release
+        uses: robinraju/release-downloader@v1
+        with:
+          tag: ${{ github.event.inputs.release }}
+          fileName: trunk-analytics-cli-aarch64-unknown-linux.tar.gz
+          extract: true
```

Comment:
This is fine, except we the runner is AMD64, so we should be using the `x86_64` release binary.

IMO, it'd be better to use [the setup script we have in our docs](https://docs.trunk.io/flaky-tests/get-started/ci-providers/otherci#example-upload-script) which is essentially just
```bash
curl -fsSLO --retry 3 https://github.com/trunk-io/analytics-cli/releases/download/${{ github.event.inputs.release }}/trunk-analytics-cli-x86_64-unknown-linux.tar.gz && chmod +x trunk
```